docs/reference/settings.rst

```diff
@@ -315,6 +315,15 @@ Specifies the number of items per page shown when viewing an image's usage (see
 
 Specifies the number of images shown per page in the image chooser modal.
 
+``WAGTAILIMAGES_RENDITION_STORAGE``
+-----------------------------------
+
+.. code-block:: python
+
+    WAGTAILIMAGES_RENDITION_STORAGE = None
```

Comment:
I think this example should contain a valid example, pretty sure setting this to `None` would cause issue, maybe use the example from your tests. `'public'`